src/clarity-angular/layout/tabs/tabs.ts

```diff
@@ -4,158 +4,45 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {
-    ContentChildren,
     Component,
-    EventEmitter,
+    ContentChildren,
     forwardRef,
-    QueryList,
-    Output, Input
+    QueryList
 } from "@angular/core";
-import {TabLink} from "./tab-link";
-import {TabContent} from "./tab-content";
-
-let nbTabsComponents: number = 0;
+import {IfActiveService} from "../../utils/conditional/if-active.service";
+import {Tab} from "./tab";
 
 @Component({
     selector: "clr-tabs",
-    templateUrl: "./tabs.html"
+    template: `
+        <ul class="nav" role="tablist">
+            <ng-content select="clr-tab"></ng-content>
+        </ul>
+        <ng-container *ngFor="let tab of tabChildren">
+            <ng-template *ngIf="tab.tabContent" [ngTemplateOutlet]="tab.tabContent.templateRef"></ng-template>
+        </ng-container>
```

Comment:
Yes. Just as a note we do the same thing for button groups. I also couldn't find a way to achieve this visually (with css) if I were to not separate out the tabContents from being inside each tab. I'm not sure what'd be a clean way to achieve this. I'll add a comment to the file so we are reminded to come back to it.